### PR TITLE
Cleanups of various Python scripts

### PR DESCRIPTION
**Pull request by practicalswift**
Fixes:
- Unused import
- Syntax error
- Redundant semicolon at end of statement
- Unused variable
- Undefined exception
- Redundant backslash
- Use of deprecated `.has_key()`
- Use of discouraged `if cond == True`
- Use of discouraged `if not x in y`
- Use of deprecated form of raising exceptions

### CODE REVIEW

**Comment by practicalswift** [comment]
The previous merge conflicts have now been resolved.

File: docs/conf.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
```
Comment (gribozavr) [inline docs/conf.py]: LGTM.

File: stdlib/public/common/MirrorCommon.py [hunk 2]
```diff
@@ -14,14 +14,14 @@
 # If you edit this, make sure to also accordingly tweak the actual template files.
 
 def getDisposition(disp=None):
-  if disp == None:
+  if disp is None:
```
Comment (gribozavr) [inline stdlib/public/common/MirrorCommon.py]: This file LGTM.

File: test/Driver/Dependencies/Inputs/fake-build-for-bitcode.py [hunk 3]
```diff
@@ -6,7 +6,6 @@
 from __future__ import print_function
 
 import os
-import shutil
```
Comment (gribozavr) [inline test/Driver/Dependencies/Inputs/fake-build-for-bitcode.py]: This file LGTM.

File: test/Driver/Dependencies/Inputs/fake-build-whole-module.py [hunk 4]
```diff
@@ -5,7 +5,6 @@
 from __future__ import print_function
 
 import os
-import shutil
```
Comment (gribozavr) [inline test/Driver/Dependencies/Inputs/fake-build-whole-module.py]: This file LGTM.

File: test/Driver/Dependencies/Inputs/modify-non-primary-files.py [hunk 5]
```diff
@@ -6,7 +6,6 @@
 from __future__ import print_function
 
 import os
-import shutil
```
Comment (gribozavr) [inline test/Driver/Dependencies/Inputs/modify-non-primary-files.py]: This file LGTM.

File: utils/GYBUnicodeDataUtils.py [hunk 6]
```diff
@@ -248,7 +248,7 @@ def create_tables(self):
         # maximum Unicode code point value is not 2^21-1 (0x1fffff), it is
         # 0x10ffff.
         self.supp_first_level_index_max = \
-            0x10ffff >> (self.supp_second_level_index_bits + \
+            0x10ffff >> (self.supp_second_level_index_bits +
```
Comment (gribozavr) [inline utils/GYBUnicodeDataUtils.py]: This file LGTM.

File: utils/SwiftBuildSupport.py [hunk 7]
```diff
@@ -187,7 +187,7 @@ def get_preset_options(substitutions, preset_file_names, preset_name):
         print_with_argv0("preset '" + preset_name + "' not found")
         sys.exit(1)
     if missing_opts:
-        print_with_argv0("missing option(s) for preset '" + preset_name + \
+        print_with_argv0("missing option(s) for preset '" + preset_name +
         "': " + ", ".join(missing_opts))
```
Comment (gribozavr) [inline utils/SwiftBuildSupport.py]: Could you adjust the indentation on this continuation line while you are here?

This file LGTM.

File: utils/apply-fixit-edits.py [hunk 8]
```diff
@@ -49,7 +48,7 @@ def apply_edits(path):
     edits_per_file = {}
     for ed in edits_set:
         fname = ed[0]
-        if not edits_per_file.has_key(fname):
+        if fname not in edits_per_file:
```
Comment (gribozavr) [inline utils/apply-fixit-edits.py]: This file LGTM.

File: utils/gyb.py [hunk 9]
```diff
@@ -478,7 +478,6 @@ def tokenGenerator(self, baseTokens):
             elif kind == 'gybLines':
                 
                 self.codeStartLine = self.posToLine(self.tokenMatch.start('gybLines'))
-                codeStartPos = self.tokenMatch.end('_indent')
```
Comment (gribozavr) [inline utils/gyb.py]: LGTM.

File: utils/line-directive [hunk 10]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
 #convert line numbers in error messages according to "line directive" comments
-import os
```
Comment (gribozavr) [inline utils/line-directive]: LGTM.

File: utils/pre-commit-benchmark [hunk 11]
```diff
@@ -14,7 +14,6 @@ import subprocess
 import sys
 import os
 import re
-import shutil
```
Comment (gribozavr) [inline utils/pre-commit-benchmark]: This file LGTM.

File: utils/sil-opt-verify-all-modules.py [hunk 12]
```diff
@@ -17,7 +17,6 @@
 import glob
 import multiprocessing
 import os
-import shutil
```
Comment (gribozavr) [inline utils/sil-opt-verify-all-modules.py]: This file LGTM.

File: utils/update-checkout [hunk 13]
```diff
@@ -68,7 +68,7 @@ def obtain_additional_swift_sources(opts = {'with_ssh': False}):
         with WorkingDirectory(SWIFT_SOURCE_ROOT):
             if not os.path.isdir(os.path.join(dir_name, ".git")):
                 print("--- Cloning '" + dir_name + "' ---")
-                if opts['with_ssh'] == True:
+                if opts['with_ssh'] is True:
```
Comment (gribozavr) [inline utils/update-checkout]: LGTM.

File: utils/viewcfg [hunk 14]
```diff
@@ -121,7 +121,7 @@ def main():
     newBlocks = { }
     for name, block in blocks.iteritems():
         for adjName in (block.preds + block.getSuccs()):
-            if not adjName in blocks:
```
Comment (gribozavr) [inline utils/viewcfg]: This file LGTM.

File: utils/swift-bench.py [hunk 15]
```diff
@@ -38,7 +38,6 @@
 
 import subprocess
 import numpy
-import time
```
Comment (gribozavr) [inline utils/swift-bench.py]: This file LGTM.

**Comment by gribozavr** [comment]
@practicalswift I don't know how to test or use the following files, so please make separate PRs for them:

```
tools/SourceKit/bindings/python/sourcekitd/capi.py
utils/cmpcodesize/cmpcodesize/compare.py
utils/pass-pipeline/*
```

File: utils/SwiftBuildSupport.py (hunk 7 above)
Comment (practicalswift) [inline utils/SwiftBuildSupport.py]: Fixed. See https://github.com/practicalswift/swift/commit/22b56d6a50d806f81d8492adcb85997f1c97f605

**Comment by practicalswift** [comment]
@gribozavr Done! 

The changes to the following files have been reverted:

```
tools/SourceKit/bindings/python/sourcekitd/capi.py
utils/cmpcodesize/cmpcodesize/compare.py
utils/pass-pipeline/*
```

Let me know if any further changes are needed :-)

**Comment by gribozavr** [comment]
Tested on Linux and OS X.

**Pull request merged**